client/python/README.md

```diff
@@ -1,19 +1,73 @@
-# Armada Python client
+Armada Python client
+=
+
+# Installation:
+
+On Linux Machine
+=
+1) Open your favorite shell, for example, good old [Bourne Again SHell, aka, bash](https://www.gnu.org/software/bash/), the somewhat newer [Z shell, aka, zsh](https://www.zsh.org/), or shiny new [fish](https://fishshell.com/).
+2) Install [Git](https://git-scm.com/) by running `sudo apt-get install git-all` on [Debian-based](https://www.debian.org/) distributions like [Ubuntu](https://ubuntu.com/), or sudo dnf install git on [Fedora](https://getfedora.org/) 
+   and closely-related [RPM-Package-Manager](https://rpm.org/) based distributions like [CentOS](https://www.centos.org/). For further information see [Installing Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+3) Fork the G-Research Armada repository to your personal repository.
+
+4) Clone the forked armada repository on your personal git repository by running git clone [Armada Repo](https://github.com/G-Research/armada.git) and navigate into the new directory Armada project by running cd armada.
+
+### Python / Pyenv and Poetry Installation
+#### Note that this repository does not support python 3.10 for now
+
+1) Install [Pyenv](https://github.com/pyenv/pyenv) on [Ubuntu](https://ubuntu.com/) and [Debian-based](https://www.debian.org/) by running `$ sudo apt-get update` , `$ sudo apt-get -y install pyenv`,  `$ sudo apt-get -y install pipenv`.
+2) Install Pyenv dependencies by running `$ sudo apt install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl git` or follow instructions [here](https://gist.github.com/jmvrbanac/8793985).
+3) Install [Pyenv](https://github.com/pyenv/pyenv) on [Fedora](https://getfedora.org/) by running `$ sudo dnf update`, `$ curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash` or follow the instructions [here](https://joepreludian.medium.com/starting-your-python-dev-environment-with-pyenv-and-pipenv-on-a-redhat-gnu-linux-based-system-d66795377ea).
```

Comment:
I'd suggest the same here - put the 3 separate commands into a single full code-block. See
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks
and note that you can specify *sh* at the beginning of the code block to get their automatic syntax highlighting.